packages/valory/services/apy_estimation/service.yaml

```diff
@@ -0,0 +1,24 @@
+name: apy_estimation
+author: valory
+version: 0.1.0
+description: A set of agents reaching consensus on APY estimates
+aea_version: '>=1.0.0, <2.0.0'
+license: Apache-2.0
+fingerprint:
+  README.md: bafybeibhbkelnxxvsln677imq65vgbglmlhyxtax4iqtzempjiwcoef3gq
+fingerprint_ignore_patterns: []
+agent: valory/apy_estimation:0.1.0:bafybeidkhhl27onhlanfcepyzhigdusb3r7cyon73c4fr34cqgvburlr4q
+number_of_agents: 4
+---
+public_id: valory/apy_estimation_abci:0.1.0
+type: skill
+models:
+  params:
+    args:
+      pair_ids:
+        uniswap_subgraph:
```

Comment:
I believe that we need to have an override for setup parameters, i.e.
safe_contract_address
apy_contract_address?

Reply:
No - there are no contracts in this app